.travis.yml

```diff
@@ -8,3 +8,6 @@ install:
     - pip install -r requirements.txt
     - pip install -r test-requirements.txt
 script: flake8
```

Comment:
Can you add the command to run the tests and generate code coverage report here?
It should be something like `coverage run -m unittest discover tests/`.